docs/reference/api/drivers/LowPowerTimer.md

```diff
@@ -2,10 +2,9 @@
 
 <span class="images">![](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_low_power_timer.png)<span>LowPowerTimer class hierarchy</span></span>
 
-LowPowerTimer inherits Timer Class, timer in this case will continue operating even in deep sleep mode. It relies on lp_ticker which is low power ticker.
+LowPowerTimer inherits from the Timer Class. The timer in this case continues operating even in deep sleep mode. It relies on `lp_ticker`, which is the low power ticker.
```

Comment:
@AnotherButler - I was not sure if we should add link to "Timer Class" here or not, your call please